### DISCUSSION TITLE
Thrust2Weight Ratio

### DISCUSSION THREAD

**Discussion by KevinHan1209** [utiasDSL/gym-pybullet-drones#222] [Q&A] [answered]
I am trying to vary the mass in simulations to prepare for domain randomization, but once I increase the weight just a little bit, the training for the HoverAviary task becomes significantly longer (I'm actually not even sure if it will converge to the target reward). This is strange, which caused me to look at the physical parameters\constants listed in the initialization of BaseAviary.py. I see that it references the thrust2weight ratio from the urdf, which is predefined. I was wondering where this value came from, as I didn't see it in any of the references documents either? Thanks.

**Comment by JacopoPan** [comment, marked as answer]
The maximum thrust mentioned on this page https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/functional-areas/pwm-to-thrust/ is of 58g, which is just a bit more than twice the weight of the CF (27g).

Of course, nothing prevents you to customize it to simulate a more powerful drone (but the controller's gains might have to be tuned accordingly)

**Comment by JacopoPan** [comment]
Hope that answers your question.
